Sources/XMTPiOS/Client.swift

```diff
@@ -155,6 +173,10 @@ public final class Client: Sendable {
 		return nil
 	}
 
+	public func canMessageV3(address: String) async throws -> Bool {
+		try await v3Client.canMessage(accountAddresses: [address]) == [true]
+	}
+
 	public static func from(bundle: PrivateKeyBundle, options: ClientOptions? = nil) async throws -> Client {
```

Comment:
It's hard to see here but did you update the from bundle method to support V3 as well?

Reply:
Ya, it's updated in the `from` that this one calls.